### DISCUSSION TITLE
Notas I1 ausentes

### DISCUSSION THREAD

**Discussion by cebabeltran** [IIC2133-PUC/2024-1#167] [Ayuda] [answered]
Hola! Tengo una duda sobre los resultados de la I1 recién publicados; están todas las pruebas revisadas o siguen en proceso? Porque me aparece todo en blanco en los puntajes, y no sé si es porque faltan pruebas por revisar o si pasó algo. Cuando voy a la página de feedback, me aparece mi prueba escaneada, pero sin ningún comentario escrito.
Muchas gracias!

**Comment by CataAlamos** [comment, marked as answer]
Hola! Tuvimos un error al subir las notas, pero debe estar todo bien ahora. En caso de que no te aparezca el feedback todavia, me puedes mandar un correo a catalina.alamos@uc.cl para solucionarlo.
